### ISSUE TITLE
some question about reproducing the result of the paper"Adaptive Universal Generalized PageRank Graph Neural Network"

### ISSUE THREAD

**Issue by Xuyike** [njuyuan/GNN_Paper_Reading#1]
Hello, sorry to trouble you. I have a quetion about the reproduction of the paper"Adaptive Universal Generalized PageRank Graph Neural Network". When I run his code, there are large differences in the results(mean_acc) of each experiment in some dataset, and the std is bigger than one or even two.I am wondering if you are in a similar situation?
Thanks for your attention.

**Comment by njuyuan** [comment]
Yes, I also met the same problem. I asked the authors about the setting of superparameters on GitHub, and the authors gave some settings shown on Github.

**Comment by Xuyike** [comment]
> Yes, I also met the same problem. I asked the authors about the setting of superparameters on GitHub, and the authors gave some settings shown on Github.

Yes, I also asked the authors, and then he showed the issues you asked. I follow his setting of superparameters, but the problem still exsits. Have you solved the problem according to the author's suggestion?

**Comment by Xuyike** [comment]
> Yes, I also met the same problem. I asked the authors about the setting of superparameters on GitHub, and the authors gave some settings shown on Github.

Yes, I also asked the authors, and then he showed the issues you asked. I follow his setting of superparameters, but the problem still exsits. Have you solved the problem according to the author's suggestion?

**Comment by njuyuan** [comment]
No

**Comment by Xuyike** [comment]
Thank you for your reply!
